runtimes/eoapi/stac/eoapi/stac/client.py

```diff
@@ -243,9 +243,35 @@ async def get_queryables(
         return queryables
 
 
+def add_render_links(collection: Collection, titiler_endpoint: str) -> Collection:
+    if renders := collection.get("renders"):
+        base_url = f"{titiler_endpoint}/collections/{collection['id']}/WebMercatorQuad"
+        for render, metadata in renders.items():
+            query_params = urlencode(metadata, doseq=True)
+            collection["links"].append(
+                {
+                    "rel": "map",
+                    "title": f"{render} interactive map",
+                    "type": MimeTypes.html.value,
+                    "href": f"{base_url}/map?{query_params}",
+                }
+            )
+            collection["links"].append(
+                {
+                    "rel": "tilejson",
```

Comment:
The issue is that if we do this, the pydantic validation won't be happy (once we turn it on) 

I think it need to be one of those https://github.com/stac-utils/stac-pydantic/blob/main/stac_pydantic/links.py#L52-L81

Reply:
we could use `tiles` for the tilejson and `preview` for the `/map` endpoint 🤷